db/models.py

```diff
@@ -1,10 +1,12 @@
 from django.db import models
 
+from django.forms import CharField
```

Comment:
The import `CharField` from `django.forms` is unnecessary and incorrect for type annotations. Consider removing it.